gorqlite_test.go

```diff
@@ -0,0 +1,74 @@
+package gorqlite_test
+
+import (
+	"log"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/rqlite/gorqlite/v2"
+)
+
+var globalConnection *gorqlite.Connection
```

Comment:
I actually liked the fact that each test controlled its table schema and data. I feel like having a global connection/state will make it hard to track down what the current state of the database is when running tests. I would personally prefer the approach that was previously implemented where each test creates its own table and tears it down afterwards.

Reply:
The problem with that is it's hard to test with `-race` flag. The current CircleCI test uses `go test -timeout 60s -v ./...`. Yet, I often uses `go test -timeout 60s -v -cover -covermode=atomic -race ./...`. If we constantly change schemas for the same table name, creating and dropping again, it doesn't make sense for that kind of test.

We might still have one connection tho, but the tables are created and dropped for each test suite, how about that?

Reply:
> We might still have one connection tho, but the tables are created and dropped for each test suite, how about that?

Yep, that sounds great.